src/NexusMods.App.UI/Pages/ModLibrary/FileOrigins/FileOriginsPageViewModel.cs

```diff
@@ -78,4 +133,68 @@ public static bool FilterDownloadAnalysisModel(DownloadAnalysis.Model model, Gam
         if (model.Contains(StreamBasedFileOriginMetadata.StreamBasedOrigin)) return false;
         return true;
     }
+
+    [UsedImplicitly]
```

Comment:
Why `UsedImplicitly`?

Reply:
Rider wasn't picking up on this in an earlier code iteration for some reason; and requested to remove it.
VS was.

That's no longer the case thankfully.